### PR TITLE
Test for Rely events in constructors + minor test re-arranging

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: test/funnels/callees/SwapperCalleeUniV3.t.sol [hunk 1]
```diff
@@ -30,6 +30,8 @@ contract SwapperCalleeUniV3Test is DssTest {
     }
 
     function testConstructor() public {
+        vm.expectEmit(true, true, true, true);
+        emit Rely(address(this));
```
Comment (sunbreak1211) [inline test/funnels/callees/SwapperCalleeUniV3.t.sol]:
```suggestion
```
This should be reverted, there is not `wards` structure in this contract.

**Review by oldchili — APPROVED** [review]
LGTM

File: test/funnels/callees/SwapperCalleeUniV3.t.sol (hunk 1 above)
Comment (telome) [reply test/funnels/callees/SwapperCalleeUniV3.t.sol]: Yes, saw it at the same time, fixed.

**Review by sunbreak1211 — APPROVED** [review]
LGTM

**Pull request merged**